src/core/expression/qgsexpressionutils.h

```diff
@@ -229,19 +229,38 @@ class CORE_EXPORT QgsExpressionUtils
       return x;
     }
 
+    static double getDoubleInternal( const QVariant &value, bool *validation )
+    {
+      const double converted = value.toDouble( validation );
+      if ( *validation == false )
+      {
+        const double localeDouble = QLocale().toDouble( value.toString(), validation );
```

Comment:
Why don't just return ?

Reply:
Must have overlooked something or didn't clean up prior versions, thank you for the review.